packages/orbit-components/src/primitives/TooltipPrimitive/components/TooltipWrapper.tsx

```diff
@@ -16,7 +16,7 @@ const TooltipWrapper = React.forwardRef<
       className={cx(
         "orbit-tooltip-wrapper",
         "max-w-full cursor-auto",
-        "focus:outline-none active:outline-none [&_:disabled]:pointer-events-none",
+        "focus:outline-offset-1 active:outline-offset-1 [&_:disabled]:pointer-events-none",
```

Comment:
Regarding the Checkbox and Radio component, after this outline change, some change is needed in those components 😬 Not sure if some adjustments should be done in this component or even if this MR or those will be fixed in those components.

Or if we should handle this outline change differently?

Reply:
If we adjust the height of this wrapper to be `h-fit-content` I believe it works well 🤔 can you confirm?

Reply:
I've tried your suggestion and it works well for the checkbox.

However, in the radio component we have double-focus and the outline styles look broken

<img width="459" alt="image" src="https://github.com/user-attachments/assets/5d7ad6e6-386a-45ff-afa4-7f403d89ee60" />

Reply:
I think the whole tooltip could be revamped. But not now. For now, I think we can accept the double focus 😕 unless you can think of something else, I'd move forward with the current solution

Reply:
@DSil How about setting outline-none for those two classes inside Checkbox and Radio?

```suggestion
        "focus:outline-offset-1 active:outline-offset-1 [&_:disabled]:pointer-events-none",
        "[&:has(.orbit-checkbox-icon-container,.orbit-radio-icon-container)]:focus:outline-none [&:has(.orbit-checkbox-icon-container,.orbit-radio-icon-container)]:active:outline-none",
```

WDYT?

Reply:
Correct. That would require some refactoring in those two components or more refactoring of Tooltip Wrapper.